README.md

```diff
@@ -107,3 +107,32 @@ Meeting Agenda is maintained [here](https://github.com/redhat-developer/service-
 Please file bug reports on [Github](https://github.com/redhat-developer/service-binding-operator/issues/new). For any other questions, reach out on `service-binding-support@redhat.com`.
 
 
+## Roadmap
+
+
+**Key**
+
+⚪️ Concept validated
+
+:white_check_mark: Works, has tests and at least one documented scenario
+
+
+
+### Maturity matrix
+
+| Feature                                        | Alpha                | Beta | GA |
+| -----------------------------------------------| -------------------- | ---- | -- |
+| Binding with Kubernetes resources              |  :white_check_mark:  |      |    |
```

Comment:
can we be more specific here? what resources exactly? Does this include support for secrets, configmap, CRs? If yes, do we need to list explicitly in the table?

Reply:
This implies "any kubernetes resource" which isn't necessarily backed by an operator.

Reply:
Do we then treat support of resource X as a separate future? Should we remove this line and list explicitelly each supported resource when gets supported?

Reply:
Unless the resource X deserves a special mention, I wouldn't list it. At the moment, we don't really restrict any resource as long as RBAC allows it.

Reply:
Binding with Kubernetes resources  = Binding of PodSpec-based workloads + Binding of non-PodSpec-based workloads , right? If yes, this line could be removed then?